File: src/test/java/com/example/scaffold/CanaryTest.java
package com.example.scaffold;

import com.example.scaffold.frameworks.test.web.IntegrationTest;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class CanaryTest extends IntegrationTest {
    @Test
    void should_be_able_to_run_spring_tests() {
        assertThat("Use-case test environment OK").isNotBlank();
    }
}

File: src/test/kotlin/com/qomolangma/CanaryTest.kt
package com.qomolangma

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class CanaryTest : IntegrationTest() {
    @Test
    fun should_be_able_to_run_spring_tests() {
        assertThat("Use-case test environment OK").isNotBlank
        QomolangmaApplication.main(arrayOf())
    }
}
